app/views/promo_mailer/new_channel_registered_2018q1.html.slim

```diff
@@ -7,7 +7,7 @@ p = t("promo_mailer.promo_activated_2018q1_verified.body_one")
 
 a.promo--item-label.promo--item-label-link-name = @channel.details_type == "YoutubeChannelDetails" ? "#{@channel.publication_title.upcase} #{t("promo.shared.on_youtube")}" : "#{@channel.publication_title.upcase}"
 .promo--referral-link-container
-  .promo--referral-link = referral_url(@channel.promo_registration.referral_code)
+  .promo--referral-link = link_to(referral_url(@channel.promo_registration.referral_code), "https://" + referral_url(@channel.promo_registration.referral_code))
```

Comment:
this is a little hacky.. should we not fix referral_url() to return the full "https://..." url?

Reply:
this method is also used on the dashboard where we do not want to display.  I could write a new method or add an argument to this one if you prefer, but we do need to differentiate somehow.

Reply:
i see. the clean way would be to have the existing display method to be `#human_referral_url`, and the complete one with HTTPS:// to be `#referral_url`. consider also a string is not actually a url if it doesn't have the protocol, which is the existing behavior.

Reply:
cool, I will write a `#human_referral_url` and push the fix.  as far as strings not being urls, i'll have to think about that a bit more..